src/firefox/remote.js

```diff
@@ -4,15 +4,53 @@ import {RemoteTempInstallNotSupported, WebExtError} from '../errors';
 import defaultFirefoxConnector from 'node-firefox-connect';
 
 const log = createLogger(__filename);
+
+
+// Flow Types
+
+import type FirefoxClient from 'firefox-client';
+
+export type FirefoxConnectorFn =
+  (port?: number) => Promise<FirefoxClient>;
+
+export type ConnectExtraParams = {
+  connectToFirefox: FirefoxConnectorFn,
+};
+
+export type FirefoxRDPAddonActor = {
+  id: string,
+  actor: string,
+};
+
+export type FirefoxRDPResponseError = {
+  error: {
+    message: string,
+  },
+};
+
+export type FirefoxRDPResponseAny = Object;
+
+export type FirefoxRDPResponseAddon = {
+  addon: FirefoxRDPAddonActor,
+};
+
+export type FirefoxRDPResponseMaybe =
+  FirefoxRDPResponseError | FirefoxRDPResponseAddon | FirefoxRDPResponseAny;
```

Comment:
Yeah, tracing the intention seems useful. It might be worth adding a comment here explaining that it exists more for documentation (otherwise, an astute reader may be tempted to correct it).